src/hyperfine/benchmark.rs

```diff
@@ -380,10 +381,11 @@ pub fn run_benchmark(
         );
 
         println!(
-            "  Range ({} … {}):   {:>8} … {:>8}    {}",
+            "  Range ({} … {}):   {:>8} … {:>8}    (median: {})  {:>10}",
             "min".cyan(),
             "max".purple(),
             min_str.cyan(),
+            median_str.blue(),
```

Comment:
This actually shows the median instead of the "max" entry and shows the "max" time after "median:"

Reply:
Oops true!